Makefile

```diff
@@ -194,22 +204,26 @@ ux-tarball: build-ux dist ## Build website asset tarball
 		--directory=src/js/engagement_view/build \
 		.
 
-.PHONY: lambdas
-lambdas: lambdas-rust lambdas-js lambdas-python ## Generate all lambda zip files
-
-.PHONY: lambdas-rust
-lambdas-rust: ## Build Rust lambda zips
-	$(DOCKER_BUILDX_BAKE) -f docker-compose.lambda-zips.rust.yml
-	docker-compose -f docker-compose.lambda-zips.rust.yml up
-
-.PHONY: lambdas-js
-lambdas-js: ## Build JS lambda zips
-	$(DOCKER_BUILDX_BAKE) -f docker-compose.lambda-zips.js.yml
-	docker-compose -f docker-compose.lambda-zips.js.yml up
-
-.PHONY: lambdas-python
-lambdas-python: ## Build Python lambda zips
-	./pants filter --target-type=python_awslambda :: | xargs ./pants package
+.PHONY: extract-lambda-zips
+extract-lambda-zips: extract-lambda-zips-js extract-lambda-zips-rust ## Extract zips from previously built Docker images for AWS Lambda
+
+.PHONY: extract-lambda-zips-rust
+extract-lambda-zips-rust: build-docker-services
+	docker run --rm \
+		--user "${UID}:${GID}" \
+		--volume "${PWD}/dist:/tmp/zips" \
+		--workdir /grapl/zips \
+		grapl/metric-forwarder-zip:${TAG} \
+		sh -c 'cp /grapl/zips/metric-forwarder.zip /tmp/zips/metric-forwarder-lambda.zip'
+
+.PHONY: extract-lambda-zips-js
+extract-lambda-zips-js: build-docker-services
+	docker run --rm \
+		--user "${UID}:${GID}" \
+		--volume "${PWD}/dist:/grapl" \
+		--workdir /grapl \
+		grapl/graphql-endpoint-zip:${TAG} \
+		cp /home/grapl/lambda.zip graphql-endpoint-lambda.zip
```

Comment:
It would be more clear to have the mountpoint map more obviously to its location on the workstation. That is, don't call it `zips` in the container, when it's `dist` outside the container. This just makes people reading the code have to do unnecessary mental gymnastics. If that calls for structuring the filesystem in the container in a different way (i.e., create a `dist` container, rather than `zips` in `/grapl`), then so be it.

Also, is there a particular reason that the `cp` commands, their invocations, and the workdirs are so different between these two `docker run` commands? To the extent possible, we should make them as identical as we can (it makes it easier to reason about, and opens the door to future abstraction and automation around this). If that requires overriding entrypoints because these have different base containers, or anything else, we should do that.

Reply:
> It would be more clear to have the mountpoint map more obviously to its location on the workstation. That is, don't call it `zips` in the container, when it's `dist` outside the container. This just makes people reading the code have to do unnecessary mental gymnastics. If that calls for structuring the filesystem in the container in a different way (i.e., create a `dist` container, rather than `zips` in `/grapl`), then so be it.

Yeah, that seems fair. I'd only ported what was defined in the compose files to docker args, so I wouldn't need to also change their respective Dockerfiles, but I agree it's better to use a common mount point.

> Also, is there a particular reason that the `cp` commands, their invocations, and the workdirs are so different between these two `docker run` commands? To the extent possible, we should make them as identical as we can (it makes it easier to reason about, and opens the door to future abstraction and automation around this). If that requires overriding entrypoints because these have different base containers, or anything else, we should do that.

The paths to copy _from_ are defined in the respective Dockerfiles, which I wasn't planning on changing in this PR. I was looking to only relieve our use of Docker Compose. I could include Dockerfile changes as well so source paths match better as well, if that's preferred.

Reply:
Yeah, I think this is a valid opportunity to make this change.